ipc/provider/src/manager/evm/manager.rs

```diff
@@ -1102,7 +1104,8 @@ impl BottomUpCheckpointRelayer for EthSubnetManager {
         let ev = contract
             .event::<lib_quorum::QuorumReachedFilter>()
             .from_block(height as u64)
-            .to_block(height as u64);
+            .to_block(height as u64)
+            .address(ValueOrArray::Value(contract.address()));
```

Comment:
Just to confirm: this won't really matter because this flow only watches a single subnet anyway: the one represented by the gateway. Right? But it's safer 👍

Reply:
It should be fine, quorum reached is emitting from the gateway of the subnet. It's not really about its children. It's just one subnet, no other subnets.